components/01-globals/variables/_variables.scss

```diff
@@ -29,8 +29,11 @@ $color-inverse         : $gray-dark !default;
 $body-bg               : $white !default;
```

Comment:
I'd added here some globals variable. In component we often use some specific color instead like `$gray` insted of for example `$border-base-color`. So I propose to assign some colors variable to global colors use in project:
1. instead of `$body-bg` create `$main-bg-color`, `$dark-bg-color` `$light-bg-color` et, accourding to our design (differents color for example in footer: 
2. create some $border-base-color - darg and light
3. create main padding and margins values
4. if you think we need others global for project, please add them :)

Reply:
in `$transition-base` and `$transition-fade` I think we need at least 0.25s - 0.3s. What do you think?

Reply:
Done :)